packages/server/src/workers/utils.ts

```diff
@@ -106,3 +107,47 @@ export async function createAuditEvent(
     outcomeDesc,
   });
 }
+
+export function isJobSuccessful(subscription: Subscription, status: number): boolean {
+  const successCodes = getExtension(
+    subscription,
+    'https://medplum.com/fhir/StructureDefinition/subscription-success-codes'
+  );
+
+  if (!successCodes?.valueString) {
+    return status >= 200 && status < 400;
```

Comment:
It looks like this line is duplicated in a few places in this function.  Can we break it out as a helper function?